index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+<section class="toolbar">
+  <nav class="toolbar logo"><img src="images/logo.png" alt=""></nav>
+  <nav class="toolbar menu"><img src="images/menu.png" alt=""></nav>
```

Comment:
- [ ] it would be better if you use the right menu icon from Figma 
- Figma icon 
![image](https://user-images.githubusercontent.com/61790587/170547450-cc290fe8-0e64-4571-a37b-7dfb29d68a85.png)
- Yours 
![image](https://user-images.githubusercontent.com/61790587/170547475-2976cd2d-30d9-45ec-98ff-f5a4670840ea.png)

Reply:
- [ ] [OPTIONAL] It will be better to use an SVG for icons to keep the quality of the image.